### PR TITLE
add some missing helm values for multicluster setup

### PR DESCRIPTION
**Pull request by DaspawnW**
**Subject**
Add missing helm values for multicluster setup

**Problem**
When executing this without the linkerd command the two variables are missing and the rendering will generate empty values.
This produces the following gateway identity, that is also used in the gateway link command to generate the link crd:

```
mirror.linkerd.io/gateway-identity: linkerd-gateway.linkerd-multicluster.serviceaccount.identity..
```

**Solution**
Add the values as defaults to the helm chart values.yaml file. If the cli is used they are overwritten by the following parameters:
* https://github.com/linkerd/linkerd2/blob/main/cli/cmd/multicluster.go#L197
* https://github.com/linkerd/linkerd2/blob/main/cli/cmd/multicluster.go#L196

### CODE REVIEW

**Comment by adleong** [comment]
Looks good!  There are some merge conflicts that need to be resolved.

**Comment by DaspawnW** [comment]
sorry rebased to main

**Review by adleong — APPROVED** [review]
Thanks for updating!

**Comment by adleong** [comment]
Arg, I just created yet another merge conflict by merging https://github.com/linkerd/linkerd2/pull/5293.  If you have a chance to resolve the conflicts (again) then go for it.  Otherwise, once another Linkerd maintainer approves this PR, we can handle the merge.  Sorry for the moving target.

**Review by kleimkuhler — APPROVED** [review]
Thanks @DaspawnW!

**Comment by kleimkuhler** [comment]
Closing this in favor of #5346 where merge conflicts have been resolved! I've add a `co-authored-by` tag to the commit.

**Pull request closed without merging**
